package stockmarket;

message Stock {
	optional string StockID = 1;
	optional string StockName = 2;
	optional float StockOpen = 3;
	optional float StockNetChange = 4;
	optional float StockChange = 5;
	optional string StockDayRange = 6;
	optional int32 StockVolume = 7;
	optional float StockPreviousClose = 8;
	optional string Stock52WKRange = 9;
	optional float Stock1YRReturn = 10;
	optional float StockYTDReturn = 11;
	optional float StockPERation = 12;
	optional float StockEarningsPS = 13;
	optional float StockMarketCap = 14;
	optional float StockSharesOutdanding = 15;
	optional float StockPrice = 16;
	optional float StockDividend = 17;
	optional string StockSector = 18;
	optional string StockIndustry = 19;
	optional string StockSubIndustry = 20;
}

message StockList {
	repeated Stock stock = 10;
}